### PR TITLE
fix: failure of building frontend image for web console deployment within vpc

### PR DESCRIPTION
**Pull request by chenhaiyun**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: src/control-plane/backend/Dockerfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ COPY src src
 COPY package.json package.json
 COPY pnpm-lock.yaml pnpm-lock.yaml
 COPY nx.json nx.json
+COPY .projenrc.js .projenrc.js
```
Comment (zxkane) [inline src/control-plane/backend/Dockerfile:9]: it's unnecessary change

**Review by zxkane — CHANGES_REQUESTED** [review]

File: src/control-plane/backend/Dockerfile (hunk 1 above)
Comment (chenhaiyun) [reply src/control-plane/backend/Dockerfile]: removed.

**Review by zxkane — APPROVED** [review]
